src/prefect/cli/server.py

```diff
@@ -293,13 +304,14 @@ def _run_in_background(
         str(keep_alive_timeout),
     ]
     logger.debug("Opening server process with command: %s", shlex.join(command))
+
+    env = {**os.environ, **server_settings, "PREFECT__SERVER_FINAL": "1"}
+    if no_services:
+        env["PREFECT__SERVER_EPHEMERAL"] = "1"
```

Comment:
I'm realizing that re-using the `ephemeral` flag means the UI will never be served properly; I'm not opposed to merging this as-is for debugging purposes, but open to push back on that - either way I'll begin thinking about how to refactor this implementation so that the UI is served alongside the webserver

Reply:
yea I should fix this prior to merging

Reply:
You could create a mapping of different server settings combinations and pass them in so these util functions remain blissfully ignorant.

Reply:
I pushed up a change - it's not beautiful by any means, but it should do the trick for now